client/signup/main.jsx

```diff
@@ -68,7 +66,6 @@ import flows from './config/flows';
 import { getStepComponent } from './config/step-components';
 import steps from './config/steps';
 import { addP2SignupClassName } from './controller';
-import SiteMockups from './site-mockup';
```

Comment:
We can also delete the `SiteMockups` component now that it's unused. And when we remove it, we can also remove:

* The `SignupSitePreview` component, its utils, and subcomponents
* The `client/lib/signup/site-theme` module

Reply:
Oh yes, you're right! I've also removed site preview related data in our codebase 👍